jquery.flot.composeImages.js

```diff
@@ -191,10 +200,17 @@
                 destination.width = Math.round(maxX - minX);
                 destination.height = Math.round(maxY - minY);
 
+                var containsSVGs = false;
                 for (i = 0; i < sources.length; i++) {
                     sources[i].xCompOffset = sources[i].genLeft - minX;
                     sources[i].yCompOffset = sources[i].genTop - minY;
+
+                    if (sources[i].srcImgTagName === 'svg') {
```

Comment:
**containsSVGs** seams to be specific to **addaptDestSizeToZoom** so it should be moved there. Also it would be nice to refactor the svg detection using **some()**.

Reply:
Done